### ISSUE TITLE
Clarify meaning of status icons in file list

### ISSUE THREAD

**Issue by corylown** [ivplus/aggregator#425]
In the organization view (for example: https://pod.stanford.edu/organizations/stanford) there is a table of contributed files that displays a status indicator:
<img width="1154" alt="Screen Shot 2022-02-03 at 9 58 02 AM" src="https://user-images.githubusercontent.com/458247/152367933-bc56d561-5a01-452a-959f-94c3f9a9f323.png">

It's unclear what these status symbols mean. It would be helpful to revise the design to provide a tool tip or legend. It may also be worth reconsidering the overall design of how these states are indicated.

The possible states are:

- File is valid MARC (`:success`  ✅)
- File of deletes (`:deletes` 🗑)
- Invalid MARC file (`:invalid` ❌ )
- File contains something other than MARC or deletes (`:not_marc` ❓)
- Unknown (`:unknown`🔄)

**Comment by ggeisler** [comment]
For the `:not_marc` and `:unknown` cases, do we just want to convey that to the user as neutral information ("here is what these are") or do we also want to draw the user's attention to them as potential problems ("hey, these are unexpected")?

In other words, is it expected and okay to have `:not_marc` and `:unknown` files in a stream, or do that statuses indicate likely file errors?

**Comment by ggeisler** [comment]
Below is a proposed update that I think will make the file status more clear. This isn't a dramatic change to anything, just:

- Adding a legend above the table to make clear what each icon that might show up in the table represents
- Replacing the current icons with icons that follow a more intentional pattern

## Example of most frequent case - valid MARC and delete files
Based on the data currently in the application, it appears that in the large majority of cases the files in a stream are all either valid MARC or a mix of valid MARC and delete files. Since this is the expected case, I think we just want the status icons to confirm to the user that the files are valid and, secondarily, help them distinguish between MARC and delete files (secondarily because both the filename and the content type fields can also potentially help the user distinguish between those two types of files). So the icons should both be green (indicating no problem with the file) but be somewhat different (to help distinguish type of file). I used the "file" icon for both cases to more closely match what the item is and to look less like a potential action button (e.g. the current trash can icon).

The example below shows the legend with all potentially displayed status icons, but just the valid MARC and deletes icons displayed in the table. This is roughly what the stream files table will look like for most organizations most of the time.

<img width="1067" alt="Screen Shot 2022-03-14 at 2 29 36 PM" src="https://user-images.githubusercontent.com/101482/158267150-b75ba71d-83ee-4549-b647-554b1fea7d55.png">

## Example of an extreme case - using all icons
A file with an invalid MARC status seems important to highlight in the table, so it gets a different icon and the `danger` color. I've used the `warning` and `info` colors for the `:not_marc` and `:unknown` statuses, but note my comment above asking how problematic files in these states actually are. We might change the color of them depending on whether these states are not common but fine to have, or if we want to draw the user's attention to them as potential problems.

To show how the less common icons would display in the table when needed:

<img width="1067" alt="Screen Shot 2022-03-14 at 2 29 53 PM" src="https://user-images.githubusercontent.com/101482/158267545-0be5b290-f107-4b6f-b3dd-cddb298ef68d.png">

## Notes
- All of these icons are from Font Awesome; I can provide pointers to them if we decide to go with this suggestion.
- I wonder if we can help the user who is checking the status of files using this table even more by making the table columns sortable? I don't know this for sure, but it seems possible that it could be helpful to be able to sort by the Content type or Records fields when using this table to check on something related to files. 
- Not really relevant to this ticket but I do have some designs to propose at some point where we remove the first row of this table (the one with the aggregated totals) and present it differently outside of the table. But that is part of a larger page reorganization discussion.

**Comment by marlo-longley** [comment]
@ggeisler I could take a look at this today. Your icon suggestions seem great to me - how would I know if they've been approved / work could start on the ticket?

I'm going to break out the question of sorting tables into another ticket / question. It's not functionality that's built in to Bootstrap - it would require writing some code in rails, so I'm not sure the priority there. I like the idea though!

**Comment by ggeisler** [comment]
@marlo-longley I don't really know what needs to be done to decide this is actionable. I'm proposing a design but especially in this project where I'm not as knowledgeable about the users and larger goals, I'm not sure I should be the one making the final call.

The proposed design solution feels fairly non-controversial to me, so maybe we don't need much consensus or discussion but perhaps at least getting @corylown to weigh in would be good.

**Comment by corylown** [comment]
@marlo-longley @ggeisler yeah -- normally I'd want some consensus from the team on the proposal, but I think this is a clear improvement and not controversial. My guess is if there are suggestions/objections they'd be small, iterative tweaks.
